lib/isorender/dom-shims.js

```diff
@@ -61,6 +61,10 @@ class HTMLElement extends Element {
     }
     return this.__observedAttrs.includes(name);
   }
+
+  attachShadow() {
+    return document.createElement(`#shadow-root`);
```

Comment:
Interesting - this works I guess because `html-element` doesn't check tag names strictly, but `#shadow-root` is an invalid tag name so this would break if the shim became more realistic (Chrome for instance throws an exception if you try this).

Reply:
I modelled it after what dev-tools showed me. Happy to change it to something else

![image](https://user-images.githubusercontent.com/1018196/54498515-b68d9500-48c5-11e9-93c2-b8dd6aadd6a5.png)

Reply:
calling it `shadow-root`